### PR TITLE
getTableList() return only tables not views

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #
follow up from https://github.com/joomla/joomla-cms/issues/42772

### Summary of Changes
exclude views from query results of `getTableList()`  same as postgres

### Testing Instructions
see https://github.com/joomla/joomla-cms/issues/42772

### CODE REVIEW

File: src/Mysql/MysqlDriver.php [hunk 1]
```diff
@@ -486,8 +486,8 @@ public function getTableList()
     {
         $this->connect();
 
-        // Set the query to get the tables statement.
-        return $this->setQuery('SHOW TABLES')->loadColumn();
+        // Set the query to get the tables statement and not the views.
```
Comment (richard67) [inline src/Mysql/MysqlDriver.php:489]: Either revert the change on this comment, or do the same change on the same comment in the other file, so both comments are consistent.

Comment (alikon) [reply src/Mysql/MysqlDriver.php:489]: done same comment

**Comment by HLeithner** [comment]
Merged as decided by the maintainer meeting.
